### DISCUSSION TITLE
Next Major .NET SDK Version

### DISCUSSION THREAD

**Discussion by saralvasquez** [braintree/braintree_dotnet#136] [Ideas]
## 👋  Welcome
We're using GitHub Discussions as a place to engage with the members of our community, to share ideas, and to support one another.

## 💡  Discussion Topic: Next Major Version of the Braintree .NET SDK
The Braintree SDK team is deciding what to include in our next major version release (v6). We are looking for some insight on:
- Language-specific improvements that you would like to see
- Features you'd like us to consider
- Pain points you'd like to see resolutions to

## 🗣 Let Us Know
Please use this thread to voice your opinions, concerns, or thoughts! We'd love to know how we can make the next version better for you.

**Comment by petcarerx** [comment]
Remove dependency on Newtonsoft.Json
